### PR TITLE
Cache data about members

### LINKED ISSUE #342: Section is not shown in admin pages
If you edit a user in the admin interface, there is no section shown, even though the user has selected a section. The same applies for programme and registration year

### LINKED ISSUE #343: Remove unnecessary fields from melos client
https://github.com/UTNkar/moore/blob/7603bde7033b3435110b0fa60a743cbb2ccd1104/src/utils/melos_client.py#L43-L44

We should remove these fields since we store email and phonenumber in our own database instead of using melos.

### LINKED ISSUE #346: Remove unnecessary fields from member model
There are many fields in the member model that no longer are used. We should go through these and remove those we don't need

### LINKED ISSUE #450: Cache the information about users
﻿Pages that have to load information about many users (contact sites, the users site in the admin interface) are very slow. This is because the system has to get the data for all those users from melos. Since the information is not updated that often, we can create a function that updates all users that have not been updated in 7 days.

The idea is that we can add fields to the member model and store the information as a part of the model. The data we need to save is the name and personnummer for the user.

### PR DESCRIPTION
**Pull request by dakaza98**
This PR introduces a "cache" for all members. Right now the system fetches all information for a user from melos which slows it down a lot. This is solved by adding two fields to the member model, name and person_nr, where the data from melos is stored. 

This data is added when a user registers or by the migration that populates the existing users. This "cache" doesn't expire since the name and person number rarely changes for a person. Instead there is a button to update your information if you wish to do so on the profile page. This decreases the load time of certain pages a lot.

Apart from that I also fixed some things on the way.

I removed some unnecessary fields from the melos client and member model. This required me to change Member to inherit `AbstractBaseUser` instead of `AbstractUser` which is why there are a bunch of new fields in the member model. They already exists in the current version of moore, now they have just been moved to our model. This closes #343 and closes #346.

I also fixed sections, programmes and registration years not showing up in wagtail when editing a user. This closes #342 

Then there is some minor translation fixes and some tests that I have added for the things I have added and some other things that needed tests.

Closes #450 

Important about this PR. When these changes are deployed to the server, the migration will take a long time since it has to collect information for every user that has an account which is about 700 people. But I have added a progress printer that says how far it's gotten. This might not be shown in ansible though. It might be possible to see it with verbose mode activated. It should also be possible to run it several times if it would fail due to timeout or some other reason

### CODE REVIEW

File: src/members/models/member.py [hunk 1]
```diff
@@ -212,18 +278,21 @@ def get_status(self):
...
+        if melos_data is not None:
+            self.name = "{} {}".format(
+                melos_data['first_name'].strip(),
+                melos_data['last_name'].strip()
+            )
+            self.person_nr = melos_data['person_number']
+            self.save()
+
+            return True
+        return False
```
Comment (Antonfogelberg) [inline src/members/models/member.py:295]: Är det här det blir sparat i cache?

**Review by Antonfogelberg — COMMENTED** [review]
Låter riktigt bra att spara i cache istället! 

Har lite frågor kring hur det sker bara. Är osäker på hur man hanterar uppdateringar utav migrations också. 

👍  Annars

File: src/members/models/member.py (hunk 1 above)
Comment (dakaza98) [reply src/members/models/member.py:295]: yes

**Comment by dakaza98** [comment]
I'm not finished with it yet though! It is still a draft

**Review by Antonfogelberg — APPROVED** [review]
+1

**Pull request merged**